### PR TITLE
dsa resources

### LINKED ISSUE #1375: [Add] Add resources for data structures and algorithms
### Site URL

https://www.youtube.com/watch?v=rZ41y93P2Qo&list=PL9gnSGHSqcnr_DxHsP7AW9ftq0AtAyYqJ&ab_channel=KunalKushwaha

### Description of the above provided site

Want to add dsa playlist by kunal kushwaha

### Category

Youtube

### Sub Category

Data structures

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by karthiknadar1204**
## Fixes 
Closes #1375

## Changes proposed
Add dsa resources in javascript

### CODE REVIEW

**Review by CBID2 — DISMISSED** [review]
LGTM

**Review by rupali-codes — DISMISSED** [review]

**Comment by rupali-codes** [comment]
please don't over-do anything. Make the changes specific to your issue. If you wanna make any other changes please raise a different issue

**Comment by CBID2** [comment]
Fix the conflicts @k-deepak04

**Comment by rupali-codes** [comment]
> Fix the conflicts @k-deepak04

it should be @karthiknadar1204 no?

**Comment by CBID2** [comment]
> > Fix the conflicts @karthiknadar1204 
> 
> 
> 
> it should be @karthiknadar1204 no?

Yeah thanks for clarifying @rupali-codes

**Comment by karthiknadar1204** [comment]
Done @rupali-codes @CBID2

**Comment by k-deepak04** [comment]
> Fix the conflicts @k-deepak04

🥲

File: database/youtube/data-structures.json [hunk 1]
```diff
@@ -121,7 +122,11 @@
   },
   {
     "name": "CODING NINJAS",
+
+    "description": "This channel provides a DSA and Python course from beginner to advanced.",
```
Comment (CBID2) [inline database/youtube/data-structures.json:126]:
```suggestion
  
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
You have the description for Coding Ninjas twice

**Comment by CBID2** [comment]
> > Fix the conflicts @karthiknadar1204 
> 
> 
> 
> 🥲

Sorry @k-deepak04

**Comment by karthiknadar1204** [comment]
I added resources for javascript by code step by step @CBID2

**Comment by karthiknadar1204** [comment]
not the one with python

**Comment by CBID2** [comment]
> not the one with python

Oh ok

**Comment by k-deepak04** [comment]
> > > Fix the conflicts @karthiknadar1204
> > 
> > 
> > 🥲
> 
> Sorry @k-deepak04

no worries
chill 💃

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by CBID2** [comment]
Now you can review @k-deepak04

**Review by k-deepak04 — APPROVED** [review]
Looks good

**Pull request merged**